src/System.Data.Common/tests/System/Data/DataSetWriteXMLTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+//
+// Copyright (C) 2004 Novell, Inc (http://www.novell.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
```

Comment:
Is this test really subject to Novell copyright? It isn't if you just wrote it?

Reply:
I simply copied the header from other test files in the repo... maybe the tests need to have open source license headers?

Reply:
This shows up on other files in this project because those files came from Mono, and when we pull source from elsewhere, we need to retain the copyright header used in the original location.

But if you're just writing code from scratch, as it sounds like you are, the header should just be:
https://github.com/dotnet/corefx/blob/9b2f7a834691a3bbad2949181eec707dd4b5a13b/src/Common/tests/System/Threading/Tasks/Sources/ManualResetValueTaskSource.cs#L1-L3

Reply:
Updated!